kvirt/config.py

```diff
@@ -655,13 +656,23 @@ def create_vm(self, name, profile, overrides={}, customprofile={}, k=None,
                         os.makedirs(destdir, exist_ok=True)
                     common.fetch("%s/%s" % (onfly, script), destdir)
                 script = os.path.expanduser(script)
-                if basedir != '.':
+                if basedir != '.' and not script.endswith('join.sh'):
                     script = '%s/%s' % (basedir, script)
+                else:
```

Comment:
modifications related to k3s should only go in the k3s directory

Reply:
Agreed .. just wasn't sure how-to implement that.